src/Symfony/Component/Security/Csrf/TokenGenerator/UriSafeTokenGenerator.php

```diff
@@ -27,6 +27,10 @@ class UriSafeTokenGenerator implements TokenGeneratorInterface
      */
     public function __construct(int $entropy = 256)
     {
+        if ($entropy <= 0) {
+            throw new \InvalidArgumentException('Entropy should be greater than 0.');
```

Comment:
It should be greater than 7, actually. ☝🏻🤓 

https://3v4l.org/KLgeS

Reply:
Done :smile: